setup.py

```diff
@@ -6,7 +6,7 @@
     packages=find_packages(exclude=('tests')),
     py_modules=['asciimath_printer', 'latex2sympy'],
     install_requires=[
-        'sympy==1.4',
+        'sympy>=1.4',
```

Comment:
do we want this to be specific still? 1.10.1 is in the requirements files?

Reply:
@lily514 yeah I guess we could, just to make sure requirements are installed correctly